components/script/dom/globalscope.rs

```diff
@@ -377,10 +376,34 @@ impl GlobalScope {
 
         // Step 9.
         if event_status == EventStatus::NotCanceled {
+            if let Some(chan) = self.devtools_chan() {
```

Comment:
We should only report the error to the devtools if we don't forward it to a worker object in the code that follows this block.